experimental/devices/ht16k33/example_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+package ht16k33_test
```

Comment:
You should keep a space between the package name and the copyright notice so it cannot be confused as a package documentation.